### ISSUE TITLE
Drag and Drop does not work on Linux Mint

### ISSUE THREAD

**Issue by yllekz** [mihaip/infinite-mac#32]
I tried dragging and dropping a file from Linux Mint's file manager (Nemo) into system7.app's main window and the file never transfers. The UI for drag/drop appears, but the actual transfer does not happen. I tried in both Firefox and Ungoogled Chromium, same result.

**Comment by mihaip** [comment]
Did you check in the "Downloads" folder? If the window is already open, you may need to close and re-open it (see #23).

Unfortunately I don't have access to Linux Mint, so it's hard for me to try to reproduce this.

**Comment by yllekz** [comment]
Yes I did, and I also saw that issue previously and attempted to close and reopen it but that didn't make it appear for me.
